### DISCUSSION TITLE
Helm chart for prowler

### DISCUSSION THREAD

**Discussion by shot4free** [prowler-cloud/prowler#4358] [General]
I have created a helm chart for prowler as we want to use it for scan of our k8s infrastructure.
If you want you can add it to your repository.

Here are the links:
repo link: https://github.com/shot4free/prowler-helm
artifacthub.io link: https://artifacthub.io/packages/helm/prowler/prowler

BR

**Comment by jfagoagas** [comment]
Hi @shot4free, how are you?

We'd love to have that in the repository, could you open a PR and place it under `contrib/k8s`? You can create a new folder within.

Thanks for using Prowler 🚀

**Comment by jfagoagas** [comment]
Thanks again for contributing with the PR https://github.com/prowler-cloud/prowler/pull/4370 and for using Prowler🚀
